route/route.go

```diff
@@ -78,7 +78,7 @@ func (r *Router) handle(handlerName string, h http.HandlerFunc) httprouter.Handl
 		defer cancel()
 
 		for _, p := range params {
-			ctx = context.WithValue(ctx, param(p.Key), p.Value)
+			ctx = context.WithValue(ctx, param(p.Key), p.Value) //nolint:fatcontext
```

Comment:
if we turn this on, only to ignore it, is it worth using?

Reply:
Fair point. It's useless in these particular instances, I couldn't find any better way of doing it. It probably has real use cases since it's enabled on Prometheus?

Reply:
being in sync with prometheus seems like a good enough reason

Reply:
Ack, someone might otherwise try to reintroduce the same linter in the future when syncing with Prometheus.